rpcs3/Emu/Cell/lv2/sys_game.cpp

```diff
@@ -2,12 +2,23 @@
 #include "util/sysinfo.hpp"
 #include "Emu/Memory/vm_ptr.h"
 #include "Emu/Cell/ErrorCodes.h"
+#include "Emu/Cell/timers.hpp"
+#include "Emu/System.h"
 #include "Emu/system_config.h"
 
+#include <thread>
+
 #include "sys_game.h"
 
 LOG_CHANNEL(sys_game);
 
+bool watchdog_stopped;
+u64 watchdog_last_clear;
```

Comment:
these should probably be atomics or something

Reply:
> these should probably be atomics or something

I agree. Could you provide some example implementation of atomics?

Reply:
you can just use:
```
atomic_t<bool> watchdog_stopped;
atomic_t<u64> watchdog_last_clear;
```

Reply:
Thanks!